src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -50,7 +50,7 @@ class UnsupportedSchemeException extends LogicException
             'class' => Bridge\Firebase\FirebaseTransportFactory::class,
             'package' => 'symfony/firebase-notifier',
         ],
-        'free-mobile' => [
+        'freemobile' => [
```

Comment:
- component name is `free-mobile-notifier`
- host is `https://smsapi.free-mobile.fr/sendmsg`

So isn't `free-mobile` as a scheme more consistent?

Though it'll probably mean a BC break and probably not worth it.

Reply:
You are right, but currently we do not have any scheme in mailer or notifier containing a - 

It would be easy to change and stay BC which is not really important, because all bridges are experimental right now.